src/main/java/com/srans/nestserver/controller/NotificationController.java

```diff
@@ -27,23 +30,30 @@
 	@Autowired
 	private NotificationRepository notificationRepo;
 	
+	@Autowired
+	private VacationRepository vacationRepo;
+	
 	
  
 	//Get Notification	
-	@GetMapping(value = "/users/{userId}/notifications")
```

Comment:
Why u renamed notifications to vacation, then what about notifications?

Reply:
I changed it, Sir